app/api/constituencies/[slug]/route.ts

```diff
@@ -21,26 +21,18 @@ export async function GET(
   request: NextRequest,
   { params }: { params: { slug: string } },
 ) {
-  console.log(`GENERATING API RESPONSE FOR CONSTITUENCY ${params.slug}`);
-
-  // The cached version of constituency data doesn't seem to work in this file - because
-  // it's an API route perhaps?
-  const constituencyData: ConstituencyData = await getConstituencyData(
-    params.slug,
-    false,
-  );
-  console.log("Successfully fetched constituencies data");
+  const constituencyData: ConstituencyData = getConstituencyData(params.slug);
 
   if (constituencyData) {
-    console.log(
+    /*    console.log(
```

Comment:
Can just delete this, was primarily just debugging when I was building out the API